config/routes.rb

```diff
@@ -493,15 +493,23 @@
   end
 
   unless Rails.env.production?
-    namespace :lti_deployment do
-      get 'get_canvas_config'
-      post 'launch'
-      get 'public_jwk'
-      post 'redirect_login'
-      get 'choose_course'
-      post 'choose_course'
-      post 'create_course'
-      post 'create_lti_grades'
+    resources :lti_deployment, only: [] do
+      collection do
+        get 'public_jwk'
+        get 'choose_course'
+        post 'choose_course'
+        post 'create_lti_grades'
```

Comment:
why is this not an assignment member route again?

Reply:
We had talked about having lti sync available for grade_entry_forms as well, and to my mind it made sense to have it be an `lti_deployment` collection route based on that. We could definitely use two different member routes though, one on assignments and one on gefs

Reply:
The only reason why I mention it is because right now you're passing in the assignment_id as an additional parameter to this route. If it was an assignment route then we get a lot of the checks that the current user is permitted (or not) to access the route for free.

I would vote to make an assignment and grade entry form route and just have them both call the same helper function.

If you want, let's discuss and we can implement it in a different PR or not depending on what we decide.